TKeazirian.HTTPServer/Router/Router.cs

```diff
@@ -6,80 +6,56 @@ namespace TKeazirian.HTTPServer.Router;
 
 public class Router
 {
-    private readonly RoutesConfig _routesConfig;
+    private readonly Routes _routes;
 
-    public Router(RoutesConfig routesConfig)
+    public Router(Routes routes)
     {
-        _routesConfig = routesConfig;
+        _routes = routes;
     }
 
-    public Response Route(Request request)
+    public Response GetResponse(Request request)
```

Comment:
GetResponse, getResponse ... a GET Response.  These are three different things. It can be confusing to keep track of.  I like that name `Route` better in this case. What does a Router do? it Routes, using a method called Route.